scripts/LSF_management_static_server.sh

```diff
@@ -34,6 +34,12 @@ host_prefix=$(hostname|cut -f1-4 -d -)
 [ -f /etc/yum.repos.d/docker-ce.repo ] && sudo rm -rf /etc/yum.repos.d/docker-ce.repo
 [ -f /etc/yum.repos.d/yum.repos.intel.com_oneapi.repo ] && sudo rm -rf /etc/yum.repos.d/yum.repos.intel.com_oneapi.repo
 
+# Temporary fix for Dynamic Server creation
+python3 -m pip install ibm-vpc==0.10.0
+python3 -m pip install ibm-cloud-networking-services ibm-cloud-sdk-core selinux
+chmod 755 -R /usr/local/lib/python3.8
+chmod 755 -R /usr/local/lib64/python3.8
```

Comment:
@alexarvinraj 

1. Is this updating to a new version of IBM packages? Or a new install?
2. I see the comments saying this is a temp fix.  What will be the permanent fix?

Reply:
Hello @go-davidng, Please see my response.

1. We are not refreshing the base image with latest LSF packages in this bug fix. 
2. We need the latest python version for the bug fix. We are taking care of that using user data temporally. This is will be taken care in the upcoming custom image/ tile refresh.